docs/Usage.md

```diff
@@ -85,7 +85,15 @@ Important points you should consider when training your networks with compressio
   - Turn off the `Dropout` layers (and similar ones like `DropConnect`) when training a network with quantization or sparsity
   - It is better to turn off additional regularization in the loss function (for example, L2 regularization via `weight_decay`) when training the network with RB sparsity, since it already imposes an L0 regularization term.
 
-#### Step 4 (optional): Export the compressed model to ONNX
+#### Step 4 (optional): Prepare for inference model.
+During training or after the training phases have been completed, you can obtain a ready-to-use model in framework native form by removing NNCF-specific features that used during the fine-training modules.
```

Comment:
Could you clarify what exactly will happen under "obtain a ready-to-use model in framework"? Some guys might think that we are preparing a model for efficient inference in a framework. For quantization, It could mean that model weights will be converted to int8, fp32 operations will be replaced by INT8 operations, FQ will be replaced Q/DQ and propagated by the model and etc.

Reply:
May be add something like:
```
But this method does not transfer model weights in low-precision format and does not trim the number of channels:
- for quantization used fakequantizer modules that emulate quantization;
- for pruning and sparsity weights fills zeroes.
```

Reply:
That used FakeQuantizer nodes instead of real converting quantizes modules and compressed weights?